README.md

```diff
@@ -114,12 +114,46 @@ salvator-xs-h3-4x2g --ENABLE_ANDROID yes`.
 
 This will require even more time and space, as Android is quite big.
 
+## Building with prebuilt graphics for DomD+DomU
+
+You have three steps to do:
+
+1. Obtain prebuilt graphics binaries from us. Pay attention that you can't use prebuilt binaries from Renesas because those packages do not support virtualization.
+2. Put obtained binaries into `<directory_with_yaml>/../prebuilt_gsx/`. By default prebuilt binaries are expected to be in the dedicated folder on the same level as your folder with yaml. But this can be changed in yaml file. Provide your directory in line
```

Comment:
Could you please wrap around longs lines? It looks really weird in wide monitors. We stick to 80 symbols per line.

Reply:
Fixed. Sorry, I forgot to wrap the lines before submitting.